build.sbt

```diff
@@ -431,9 +444,20 @@ lazy val unidocSettings = Seq(
   ),
   JavaUnidoc / unidoc / unidocAllSources := (JavaUnidoc / unidoc / unidocAllSources).value.map {
     sources =>
-      sources.filterNot { source =>
-        source.toString.contains("$") || source.toString.contains("packageprivate")
-      }
+      sources
+        .filterNot { source =>
+          source.toString.contains("$") || source.toString.contains("packageprivate")
+        }
+        .filter { source =>
+          //
+          // This filter limits javadoc to only files with
+          // api, sapi, or japi directories in their paths.
+          //
+          val str = source.toString
+          str.contains("/api/") ||
+          str.contains("/sapi/") ||
+          str.contains("/japi/")
+        }
```

Comment:
I believe this will exclude the UDFs , as they don't have api in their path. Is that intentional?

Reply:
Unintentional.